React/Views/RCTTextField.m

```diff
@@ -42,6 +42,25 @@ - (void)setText:(NSString *)text
   }
 }
 
+- (void)_setupPlaceholder {
```

Comment:
@bimawa - what happens here if the placeholder is removed by setting the prop to null / empty string?